conans/model/ref.py

```diff
@@ -61,25 +61,29 @@ class ConanFileReference(namedtuple("ConanFileReference", "name version user cha
     sep_pattern = re.compile("@|/|#")
     revision = None
 
-    def __new__(cls, name, version, user, channel, revision=None):
+    def __new__(cls, name, version, user, channel, revision=None, validate=True):
```

Comment:
Should `validate` defaulted to False and only validate in the ConanFileReference loading points?

Reply:
I feel safer if we keep the previous behavior (always validating), at least in this pull request. Afterward, we can force it to be explicit and decide at each point if validation is needed or not.